addons/crm/tests/test_crm_lead.py

```diff
@@ -81,3 +82,25 @@ def test_mailgateway(self):
         new_lead.handle_partner_assignment(create_missing=True)
         self.assertEqual(new_lead.partner_id.email, 'unknown.sender@test.example.com')
         self.assertEqual(new_lead.partner_id.team_id, self.sales_team_1)
+
+    @mute_logger('odoo.addons.phone_validation.tools.phone_validation')
+    def test_field_email_and_phone(self):
+        partner = self.env['res.partner'].search([('email', '!=', False), ('phone', '!=', False)], limit=1)
```

Comment:
There are already existing data for testsz. Don't do fuzzy searches.

Reply:
In CrmCommon there's no lead with partner (only possible with `_create_duplicates` so I'll add one in `setUpClass`)

Reply:
take lead_1 and just manually set partner_id to False . That way we work on a single record that we pre-setup in some tests :)